modules/rostests/apitests/advapi32/testlist.c

```diff
@@ -7,6 +7,7 @@ extern void func_CreateService(void);
 extern void func_DuplicateTokenEx(void);
 extern void func_eventlog(void);
 extern void func_HKEY_CLASSES_ROOT(void);
+extern void func_Hash(void);
```

Comment:
usually we sort these functions by alphabetical order (thus that one should be before the hkey one)

Reply:
Committed. I had thought 'K' < 'a'...